django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -87,13 +87,41 @@
         }
     }
 
+    function updateRelatedSelectsOptions(currentSelect, win, objId, newRepr, newId) {
+        // After create/edit a Model from the options next to the current SELECT (+ or :pencil:)
+        // update ForeignKey PK of the rest of SELECTs in the page
+
+        const path = win.location.pathname;
+        // we have to extract the model from the popup url
+        // '.../<model>/add/' or '.../<model>/<id>/change/'
+        // depending the action (ADD or CHANGE)
+        const modelName = path.split('/')[path.split('/').length - (objId ? 4 : 3)];
+        const selectsRelated = document.querySelectorAll(`[data-model-ref="${modelName}"] select`);
+
+        selectsRelated.forEach(function(select) {
+            if (currentSelect !== select) {
```

Comment:
Reverse the condition and [return early](https://szymonkrajewski.pl/why-should-you-return-early/) to decrease the indentation level. It will be easy to read.

Reply:
Totally agree! Thanks for the link!